Vagrantfile

```diff
@@ -135,6 +135,10 @@ set -o pipefail
 # something that we don't want to happen when running provision in a
 # development environment not using Vagrant.
 
+# Set up connection inside container
+IP=$(/sbin/ip route | awk '/defualt/ { print $3}')
+ip route add default via $IP
```

Comment:
So you’re adding a default route to the IP address that’s already the default route? Or you would be, if `defualt` weren’t mispelled?

I suspect this didn’t really fix anything for you, and instead you were seeing a random network failure that randomly went away.

Reply:
Sorry for the typo. I am adding this as I suspect there may be an issue with the vagrant machine while assigning the default route though not much sure about it.

Actually, there is an issue with the vagrant machine due to which it fails to connect to the internet even without any changes to any file. I had read about this fix in this [thread](https://github.com/hashicorp/vagrant/issues/8234). Since this thing seems to work from the last couple of comments, I thought of implementing it and test it.

Reply:
Ah, just tested now. It throws another error which does not seems to fix the issue. Sorry for the wrong PR. Should I close this PR?

Reply:
No problem; thanks for the follow-up. If you have more details (e.g. specific error messages), can you add them to #14300?

Reply:
Sure, I will add them. Thanks @andersk